### PR TITLE
Disallow 2-plane 444 formats and fix typeSize for _nPACKxx suffix formats.

### LINKED ISSUE #202: Disallow two-plane 444 formats
The [`VK_EXT_ycbcr_2plane_444_formats`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_ycbcr_2plane_444_formats.html) extension (promoted to Vulkan 1.3) added four new multiplanar formats so the KTX spec should be updated to disallow them.

### LINKED ISSUE #203: typeSize spec for formats with suffix _nPACKxx is wrong
It says typeSize must be n * xx / 8. However it should just be xx / 8 as 'n' here means there are n of the PACKxx quantities and it is the PACKxx quantities will that need swapping.

### PR DESCRIPTION
**Pull request by MarkCallow**
Fixes #202. Fixes #203.

### CODE REVIEW

**Comment by MarkCallow** [comment]
@lexaknyazev, @aqnuep please review and let me know if there is anything else we need to fix before submitting to the WG for approval.

File: ktxspec.adoc
Comment (lexaknyazev) [inline ktxspec.adoc:2437]: Shouldn't the field name and the format name pattern be wrapped with backticks?

**Review by lexaknyazev — APPROVED** [review]

**Pull request merged**
